framework/source/class/qx/util/ResourceManager.js

```diff
@@ -283,16 +283,31 @@ qx.Class.define("qx.util.ResourceManager",
             continue;
           }
 
+          var href;
+          //first check if there is base url set
+          var baseElements = qxWeb("base");
```

Comment:
could you use `qx.bom.Selector.query('base',Document)` I would rather not add a `qxWeb` dependency at this point